src/features/medias/media.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express'
+import { IStorageProvider } from '@/providers/interfaces/storage.interface'
+import { MediaService } from './media.service'
+import { i18n } from '@/loaders/i18n'
```

Comment:
Could you please explain the benefits of using this internal loader?

Reply:
the loaders folder? I add everything that's needed to be loaded on the project setup, like everything that's is important we are loading/adding into memory to be available to use